release-notes/release-notes5101.md

```diff
@@ -0,0 +1,153 @@
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+
+### Security related dependency update for all 3 server-side services
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+
+### Fixed Link Checker highlighting for unresolvable hosts
+
+Invalid links for unresolvable (non-existent) hosts are correctly identified and highlighted in the editor.
+
+### Updating the self-hosted server-side components
+
+The new versions of the server-side services provide updates for the Java-based server-side components. To deploy the updated version of the server-side components:
+
+1. Update your Java Application Server to the minimum required version:
+
+    - Eclipse Jetty 9.4 or later
+    - Apache Tomcat:
+        - 9 or later
+        - 8.5.12+
+        - 8.0.42+
+        - 7.0.76+
+
+2. Replace the existing server-side `.war` files with the `.war` files bundled with {{site.productname}} 5.10.1 or later.
+
+For information on:
+
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+- Deploying the server-side components using Docker, see: [Containerized service deployments]({{site.baseurl}}/enterprise/server/dockerservices/).
+
+## General bug fixes
+
+{{site.productname}} 5.10.1 provides fixes for the following bugs:
+
+- The iframe aria help text was not read by some screen readers.
+- Clicking the `forecolor` or `backcolor` toolbar buttons would do nothing until selecting a color.
+- Crop functionality did not work in the `imagetools` plugin when the editor was rendered in a shadow root.
+- Fixed an exception thrown on Safari when closing the `searchreplace` plugin dialog.
+- The `autolink` plugin did not convert URLs to links when starting with a bracket.
+- The `autolink` plugin incorrectly created nested links in some cases.
+- Tables could have an incorrect height set on rows when rendered outside of the editor.
+- In certain circumstances, the table of contents plugin would incorrectly add an extra empty list item.
+- The insert table grid menu displayed an incorrect size when re-opening the grid.
+- The word count plugin was treating the zero width space character (`&#8203;`) as a word.
+
+## Security fixes
+
+{{site.productname}} 5.10.1 provides fixes for the following security issues:
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes _High_ severity vulnerability fixes.
+
+For information on the server-side components updates, see: [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges).
+
+## Known issues
+
+This section describes issues that users of {{site.productname}} 5.10.1 may encounter and possible workarounds for these issues.
+
+- [](#)
```

Comment:
We need these placeholders removed on the final merge

Reply:
The known issues section has been removed now.